src/assets/scripts/client/base/BaseModel.js

```diff
@@ -8,15 +8,15 @@ import _uniqueId from 'lodash/uniqueId';
  * @class BaseModel
  */
 export default class BaseModel {
-    constructor() {
+    constructor(modelName = 'BaseModel') {
         /**
          * Unigue string id that can be used to differentiate this model instance from another.
          *
          * @property _id
          * @type {string}
          * @private
          */
-        this._id = _uniqueId();
+        this._id = this.verifyModelName(modelName) + _uniqueId();
```

Comment:
additionally, `_uniqueId` accepts a string that can be used as the id base.  so in our case: 
```
_uniqueId(`${modelName}-`)
```